posts/views.py

```diff
@@ -41,13 +44,37 @@ def put(self, request):
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
 
+class AdminUserView(TemplateView):
+    @method_decorator(login_required)
+    def get(self, request, *args, **kwargs):
+        if not request.user.is_staff:
+            raise PermissionDenied
+        unapproved = User.objects.filter(approved=False)
+
+        return render(request, 'users/approve_user.html', context={'unapproved': unapproved})
+
+    @method_decorator(login_required)
+    def post(self, request):
+        if not request.user.is_staff:
+            raise PermissionDenied
+        user_to_approve = request.POST['user']
```

Comment:
is 'user' supposed to be 'id' like in the javascript post?

Reply:
The javascript isn't used. It gets the post value from the form created in the template. Normally that woud be right, I removed the js file to make it more clear.